test/boulder-tools/tag_and_upload.sh

```diff
@@ -12,11 +12,11 @@ DOCKER_REPO="letsencrypt/boulder-tools"
 # .github/workflows/release.yml,
 # .github/workflows/try-release.yml if appropriate,
 # and .github/workflows/boulder-ci.yml with the new container tag.
-GO_CI_VERSIONS=( "1.20.8" "1.21.1" )
+GO_CI_VERSIONS=( "1.21.1" )
 # These versions are built for both platforms that boulder devs use.
 # When updating GO_DEV_VERSIONS, please also update
 # ../../docker-compose.yml's default Go version.
-GO_DEV_VERSIONS=( "1.20.8" )
+GO_DEV_VERSIONS=( "1.21.1" )
```

Comment:
I believe we will need to delete all of the images and push new ones before this is merged. We probably should have caught this when we reviewed the change that skipped ARM for GO_DEV_VERSIONS.

Reply:
I'm happy to do this for you since it's relatively quick on my machine, let me know.

Reply:
Or I could just rerun tag_and_upload, since they'll get tagged with today's date, right?

Reply:
🤦🏻‍♀️ yep, for some reason I thought I pushed the images (yesterday) today. It has been a long week. Ironic, given that Monday was a holiday.